pkg/storage/tpr/storage_interface_test.go

```diff
@@ -17,12 +17,70 @@ limitations under the License.
 package tpr
 
 import (
+	"context"
+	"reflect"
 	"testing"
 
 	"github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog"
+	_ "github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/install"
+	"github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/testapi"
+	"github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1alpha1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/diff"
 )
 
+const (
+	namespace = "testns"
+	name      = "testthing"
+)
+
+func TestCreate(t *testing.T) {
+	iface, fakeCl := getTPRStorageIFaceAndFakeCoreRESTClient(t)
+	broker := &v1alpha1.Broker{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: v1alpha1.SchemeGroupVersion.String(),
+			Kind:       ServiceBrokerKind.String(),
+		},
+		ObjectMeta: metav1.ObjectMeta{Name: name},
+	}
+	outBroker := &v1alpha1.Broker{}
+	if err := iface.Create(
```

Comment:
In this context I do not find this if inlined helpful syntactically, I'd rather see:
err := ...
if err != nil 

but, your call.